DOCSP-38042: CMAP Tracking Example (#891)
mongodb/docs-node: source/fundamentals/monitoring/connection-monitoring.txt

The following sections demonstrate how to record connection pool events in your

> application or want to explore the information provided in these events.

Reviewer: S: out of scope for this ticket so not a necessary change, but this sentence is confusing. I think this wording makes more sense:
```suggestion
application and explore the information provided in these events.
```

Author: Good eye!

.. literalinclude:: /code-snippets/monitoring/cpm-subscribe.js
   :language: javascript

Connection pool monitoring events can aid you in debugging and understanding

> the behavior of your application's connection pool. The following example
> demonstrates a function that uses connection pool monitoring events to return a count
> of checked-out connections in the pool.

Reviewer: S: I think this sentence could be a little more concise, and it also can end with a colon. Maybe something like:
```suggestion
the behavior of your application's connection pool. The following example uses connection pool monitoring events to return a count
of checked-out connections in the pool:
```
